crest/Assets/Crest/Crest/Shaders/OceanInputs/AnimWavesScaleByFactor.shader

```diff
@@ -0,0 +1,123 @@
+﻿// Crest Ocean System
+
+// This file is subject to the MIT License as seen in the root of this folder structure (LICENSE)
+
+// This writes straight into the displacement texture and sets the water height to the y value of the geometry.
+
+Shader "Crest/Inputs/Animated Waves/Scale By Factor"
+{
+	Properties
+	{
+		[Enum(ColorWriteMask)] _ColorWriteMask("Color Write Mask", Int) = 15
+
+		// Scale the waves. Zero is no waves and one leaves waves untouched.
+		_Scale("Scale", Range(0, 1)) = 0.35
+
+		// Use the texture instead of the scale value.
+		[Toggle] _Texture("Apply Texture", Float) = 0
+		_MainTex("Texture", 2D) = "black" {}
+
+		// Inverts the scale value.
+		[Toggle] _Invert("Invert", Float) = 0
+
+		[Header(Feather)]
+		// Feather the edges of the mesh using the texture coordinates. Easiest to understand with a plane.
+		[Toggle] _FeatherAtUVExtents("Feather At UV Extents", Float) = 0
+		// How far from edge to feather.
+		_FeatherWidth("Feather Width", Range(0.001, 0.5)) = 0.1
+	}
+
+	SubShader
+	{
+		Pass
+		{
+			Blend Zero SrcColor
```

Comment:
Is this multiply? I guess it must be of it's working, just my brain not working right now!

Reply:
It is. I have added a comment.